src/gui/widgets/FadeButton.cpp

```diff
@@ -55,6 +55,11 @@ FadeButton::~FadeButton()
 {
 }
 
+void FadeButton::setActiveColor( const QColor & activated_color )
```

Comment:
FYI - Just noticed `activated_color` should probably be `activatedColor`.  Since the rest of the var names are relatively similar we'll fix this at a later time.